Competition2018/src/main/java/competition/subsystems/climberdeploy/ClimberDeploySubsystem.java

```diff
@@ -32,20 +37,45 @@ public ClimberDeploySubsystem(CommonLibFactory clf, XPropertyManager propMan, El
     private void initializeMotor() {
         motor = clf.createCANTalon(contract.getClimbDeployMaster().channel);
         motor.setInverted(contract.getClimbDeployMaster().inverted);
+        
+        motor.configSelectedFeedbackSensor(FeedbackDevice.CTRE_MagEncoder_Relative, 0, 0);
+    }
+    
+    public int getTicks() {
+        return motor.getSelectedSensorPosition(0);
+    }
+    
+    public double percentExtended() {
+        return getTicks() / absoluteMaxTicks.get();
     }
 
     /**
      * extends the climber arm
      */
     public void extendClimberArm() {
-        motor.simpleSet(deploySpeed.get());
+        setPower(deploySpeed.get());
     }
 
     /**
      * detracts the climber arm
      */
     public void retractClimberArm() {
-        motor.simpleSet(-deploySpeed.get());
+        setPower(-deploySpeed.get());
+    }
+    
+    
+    // Use this once we have a good understanding of limits
+    public void setPower(double power) {
+        /*if (getTicks() < 1000) {
```

Comment:
I realized that we need to have no lower limit, since we can never retract this manually. Annoying.

Eventually we will need a separate "override" command, and the regular one will be safe.

Reply:
Added post-vernon items for this